plugins/azure-functions/src/hooks/useServiceEntityAnnotations.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+
+export const AZURE_FUNCTIONS_ANNOTATION = 'portal.azure.com/functions-name';
+export const useServiceEntityAnnotations = (entity: Entity) => {
+  const functionsName =
+    entity?.metadata.annotations?.[AZURE_FUNCTIONS_ANNOTATION] ?? '';
+  const projectName =
+    entity?.metadata.annotations?.['dev.azure.com/project-repo'] ?? '';
```

Comment:
I'm not sure why this is here, maybe from initial development and can be removed?